arbitrum-docs/launch-orbit-chain/concepts/arbos-upgrade.md

```diff
@@ -0,0 +1,35 @@
+
+To upgrade ArbOS on an Orbit chain, follow these steps:
+
+1. **Wasm Module Root Upgrade**: The `WASM module root` is a 32-byte hash, created from the Merkelized Go replay binary and its dependencies. When ArbOS is upgraded, a new Wasm module root is generated due to modifications in the State Transition Function. This new wasm module root must be set in the rollup contract on the parent chain. To implement the upgrade, you should utilize the [setWasmModuleRoot](https://github.com/OffchainLabs/nitro-contracts/blob/38a70a5e14f8b52478eb5db08e7551a82ced14fe/src/rollup/RollupAdminLogic.sol#L321) method by calling the `Rollup proxy` contract. As the `upgrade executor` contract on the parent chain is the designated owner of the rollup contract, it is necessary for the **chain owner account** to initiate a call to the upgrade executor. This call should include the correct calldata for setting the new wasm module root through the setWasmModuleRoot function within the rollup contract. For example, the wasm module root for ArbOS 11 is `0xf4389b835497a910d7ba3ebfb77aa93da985634f3c052de1290360635be40c4a`. 
+**Note** that wasm module roots are backward compatible, meaning upgrading it prior to an ArbOS upgrade won't disrupt the system.
+
+1. **Schedule ArbOS Upgrade**: To update the ArbOS version, you must schedule an upgrade on the Orbit chain. This is done by calling the [scheduleArbOSUpgrade](https://github.com/OffchainLabs/nitro-contracts/blob/acb0ef919cce9f41da531f8dab1b0b31d9860dcb/src/precompiles/ArbOwner.sol#L61) function on the ArbOwner precompile of the Orbit chain. This function requires two inputs:
+    1. **newVersion**: Specify the ArbOS version you wish to upgrade to.
+    2. **timestamp**: Set the exact timestamp at which you want your Orbit chain to transition to the new ArbOS version.
+
+    As with the parent chain, performing **admin actions** on the orbit chain necessitates using the `upgrade executor` contract on the orbit chain. The **chain owner account** must execute a call to the upgrade executor with the appropriate calldata, to invoke the `scheduleArbOSUpgrade` function of the ArbOwner precompile, thereby setting the new ArbOS version and the desired timestamp for the upgrade. 
+
+3. **Update Arb Nodes**: The final step in upgrading ArbOS is to update the nodes of your Orbit chain to a version that supports the new ArbOS. This update should be performed after completing steps **1** and **2**, and prior to the **timestamp** set for the ArbOS upgrade. For example, if you are upgrading to ArbOS 11, the corresponding node version is `v2.2.2`. The Docker image for this node version is `offchainlabs/nitro-node:v2.2.2-8f33fea`. This step ensures that your nodes are compatible with the new ArbOS version and are ready to operate effectively after the upgrade.
```

Comment:
Nodes need to be fully backwards compatible with all arbos version the chain has run on so I think having the recommendation be to first issue the node release before anything else, but that people only need to actually make sure they've upgraded when the upgrade is activated.

Basically it's better to start by releasing the node upgrade to maximize how much time people have to upgrade

Reply:
So you mean changing the orders and put update arb nodes as first step @hkalodner ?